includes/class-flizpay-activator.php

```diff
@@ -44,6 +44,6 @@ public static function activate()
 
         $api_client = WC_Flizpay_API::get_instance($api_key);
 
-        $api_client->dispatch('save_webhook_url', array("woocommerceActive" => true), false);
+        $api_client->dispatch('save_webhook_url', array("isActive" => true), false);

```

Comment:
Since we're touching this, can you maybe edit the name of this action in the API file to be more meaningfull? 
And then just replaced it where it's being referenced.

```suggestion
        $api_client->dispatch('edit_business', array("isActive" => true), false);

```